### PR TITLE
Update Akka gRPC to 0.8.0

### PR DESCRIPTION
**Pull request by viktorklang**
Unfortunately running into https://github.com/akka/akka-grpc/issues/628 now

### CODE REVIEW

File: proxy/core/src/main/scala/io/cloudstate/proxy/Serve.scala [hunk 1]
```diff
@@ -230,19 +243,22 @@ object Serve {
             handler.flow
           }
 
-        val responseCodec = Codecs.negotiate(req)
+        val responseWriter = GrpcProtocolNative.newWriter(Codecs.negotiate(req))
+        val reader = GrpcProtocolNative.newReader(
+          Codecs.detect(req).getOrElse(throw new IllegalArgumentException(s"Unknown codec"))
```
Comment (viktorklang) [inline proxy/core/src/main/scala/io/cloudstate/proxy/Serve.scala]: @raboof Did you intend to include the actual codec in the exception message?

File: proxy/core/src/main/scala/io/cloudstate/proxy/HttpApi.scala [hunk 2]
```diff
@@ -442,9 +441,13 @@ object HttpApi {
         method = HttpMethods.POST,
         uri = Uri(path = Path / methDesc.getService.getFullName / methDesc.getName),
         headers = req.headers :+ IdentityHeader,
-        entity = HttpEntity(
+        entity = HttpEntity.Chunked(
           ContentTypes.`application/grpc+proto`,
-          Grpc.encodeFrame(Grpc.notCompressed, ByteString.fromArrayUnsafe(message.toByteArray))
+          Source.single(
+            GrpcProtocolNative
+              .newWriter(Identity)
```
Comment (viktorklang) [inline proxy/core/src/main/scala/io/cloudstate/proxy/HttpApi.scala]: How expensive are these new methods/API?

**Comment by raboof** [comment]
This is green on travis now, depending on a published post-0.8.4 Akka gRPC snapshot.

**Comment by viktorklang** [comment]
Thanks @raboof!

**Pull request merged**